services/ui-src/src/utils/other/styleAsOptional.ts

```diff
@@ -0,0 +1,4 @@
+import { parseCustomHtml } from "./parsing";
```

Comment:
Request: Let's move this utility method out of `other/styleAsOptional` and put it in `parsing/parsing.ts` since that's what it's doing.